scalameta/semantic/shared/src/main/scala/scala/meta/internal/semantic/codecs/package.scala

```diff
@@ -0,0 +1,86 @@
+package scala.meta.internal.semantic
+
+import org.scalameta.unreachable
+import scala.meta.io._
+import scala.meta.{semantic => m}
+import scala.meta.internal.semantic.{proto => p}
+
+package object codecs {
```

Comment:
We'll have to discuss #799 separately and come up with a fix in a different PR. In this pull request, I only plan to rename the package, nothing beyond that.